### ISSUE TITLE
Missing library libhandy

### ISSUE THREAD

**Issue by floers** [rust-lang/crates-build-env#71]
Building a crate with a dependency to the libhandy crate fails on docs.rs: https://docs.rs/crate/gstore/0.2.4/builds/358169

**Affected crates:** gstore
**Ubuntu 20.04 packages to install:** libhandy-1-0

**Comment by jyn514** [comment]
@floers are you interested in adding it yourself? https://forge.rust-lang.org/docs-rs/add-dependencies.html

**Comment by floers** [comment]
Actually I am! I will try it this weekend :)

**Comment by floers** [comment]
I just realized the libhandy package is not available in 20.04 focal which the dockerfile is based on. I guess that means I have to wait for the next LTS release?

**Comment by jyn514** [comment]
Yes, unfortunately. You could try changing your build.rs so it doesn't link to libhandy, or vendoring it and building from source.

**Comment by floers** [comment]
Ok I'll try to build it from source. Thank you for your help :). Should I close this issue now? Is there a place to keep track of libraries to add when the docs.rs machine is upgraded?

**Comment by jyn514** [comment]
This issue works fine to track upgrades that need a new OS version.

**Comment by jyn514** [comment]
I think this should be possible after https://github.com/rust-lang/crates-build-env/pull/120 :)

**Comment by syphar** [comment]
closing this, assuming it's not needed any more. 

we're on 24.04 now, https://github.com/rust-lang/crates-build-env/pull/168 so adding should be easy
